README.md

```diff
@@ -30,10 +32,20 @@ config :sentry,
 ```
 
 ### Capture Exceptions
+
+Sometimes you want to capture specific exceptions, to do so use the `Sentry.capture_exception/3`.
+
 ```elixir
-Sentry.capture_exception(my_exception)
+try do
+  ThisWillError.reall()
+rescue
+  my_exception ->
+    Sentry.capture_exception(my_exception)
```

Comment:
should add `System.stacktrace()` to the second arg, and maybe some optional stuff in the last arg to show a fuller example?